users/migrations/0001_initial.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 4.0.3 on 2022-03-24 18:53
+
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('is_superuser', models.BooleanField(default=False,
+                                                     help_text='Designates that this user has all permissions ' +
+                                                     'without explicitly assigning them.',
```

Comment:
I think it should be like:
```suggestion
                ('is_superuser', models.BooleanField(default=False,
                                                     help_text='Designates that this user has all permissions',
```

Reply:
Done. I have used parenthesis as you requested in another comment.